content/en/docs/tutorials/services/source-ip.md

```diff
@@ -48,6 +48,10 @@ the target localization.
 The examples use a small nginx webserver that echoes back the source
 IP of requests it receives through an HTTP header. You can create it as follows:
 
+{{< note >}}
+The example image below works only with AMD64 architecture.
```

Comment:
Hi @haripriya9647 .
I have not confirmed if this statement is true, but does it make sense to say something like:

The image in the following command only runs on AMD64 architectures.
or

In the following command, the image only runs on AMD64 hosts.

Reply:
@kbhawkey thank you for the review and the below line makes sense to me, I am adding the commit according to your suggestion.
"The image in the following command only runs on AMD64 architectures"

Thank you